src/app/components/Month.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+import { useContext } from 'react';
+import { AuthContext, IAppContext } from '@/src/app/auth.context';
+import { IIncome, IMonth } from '@/src/types';
+
+export default function Month(): JSX.Element {
+  const { user, userMonths, isLoadingContext } = useContext(AuthContext) as IAppContext;
+
+  return (
+    <section>
+      <h1>Month Section</h1>
+      {userMonths.length === 0 && !isLoadingContext && <p>No months to display</p>}
+      {userMonths.length > 0 &&
```

Comment:
If `userMonths` is undefined it will throw an error

Reply:
`userMonths` will always be an array.